### DISCUSSION TITLE
If an individual starts to talk, can whisper detect that another person started talking?

### DISCUSSION THREAD

**Discussion by AlphaQuantum** [openai/whisper#705] [Q&A]
let's have this scenario, 2 people are doing a podcast, can it detect when one starts to speak and the other stops?

**Comment by glangford** [comment]
No. But speaker diarization in general is a popular discussion and there is work going on in the community on this problem. See #104 #264 #273 #585

**Comment by Majdoddin** [comment]
Hi @AlphaQuantum, 
[www.lexicaps.com](http://www.lexicaps.com/) seamlessly adds diarization to Whispers transcription. No 3rd party packages.
Feel free to contact me if you need more features.
Announcement: https://github.com/openai/whisper/discussions/1537
Repo: https://github.com/Majdoddin/lexicaps
